### DISCUSSION TITLE
problems merging previous cut file

### DISCUSSION THREAD

**Discussion by jannesvdh** [mifi/lossless-cut#1327] [Q&A]
I first remove a part of a h264 AVC UHD ± 100 mbit videofile.
When I try to merge this file with other videofiles with the same format, the new craeted file is corrupted and won't play. I don't get an error message.
Tried it with different files, all with the same result. 
My present version is 3.46.2 but it happend also with previous versions.

**Comment by mifi** [comment]
Could you post info about each file? Help menu -> report

that will help us diagnose whether there are really some differences between the two files

**Comment by jannesvdh** [comment]
Made a testcase with some short videofiles. problem stays. files(total ± 200 MB) are to big to attach
sent you 3 mediainfo files and a pdf with screenprints of the merge operation. Strange is that the files are in MP4 and after merge it is MOV file. If you need the files please reply. Maybe I can send them with wetransfer or sent it to a server of yours.

[mediainfo 03b Landschap zonder grenzen deel 2.txt](https://github.com/mifi/lossless-cut/files/9674875/mediainfo.03b.Landschap.zonder.grenzen.deel.2.txt)
[mediainfo 03b Landschap zonder grenzen deel 2merged.txt](https://github.com/mifi/lossless-cut/files/9674876/mediainfo.03b.Landschap.zonder.grenzen.deel.2merged.txt)
[problem losslesscut screenprints.pdf](https://github.com/mifi/lossless-cut/files/9674877/problem.losslesscut.screenprints.pdf)
[Mediainfo 00b foto JB-1.txt](https://github.com/mifi/lossless-cut/files/9674879/Mediainfo.00b.foto.JB-1.txt)
[Mediainfo 00b foto JB-1.txt](https://github.com/mifi/lossless-cut/files/9674907/Mediainfo.00b.foto.JB-1.txt)
[mediainfo 03b Landschap zonder grenzen deel 2.txt](https://github.com/mifi/lossless-cut/files/9674908/mediainfo.03b.Landschap.zonder.grenzen.deel.2.txt)
[mediainfo 03b Landschap zonder grenzen deel 2merged.txt](https://github.com/mifi/lossless-cut/files/9674910/mediainfo.03b.Landschap.zonder.grenzen.deel.2merged.txt)
[problem losslesscut screenprints.pdf](https://github.com/mifi/lossless-cut/files/9674912/problem.losslesscut.screenprints.pdf)

**Reply by mifi** [reply]
you can share the files and send it private to me to finstaden@gmail.com if you'd like

**Reply by mifi** [reply]
Thanks for sending over. Unfortunately your files do not have identical codec parameters. If you enable the "Check files" checkbox in the Merge dialog, you will see that there are multiple issues / differences between the two files, making them incompatible for lossless merging

I will improve the UI to make this more clear.
